src/app/interfaces.ts

```diff
@@ -1,6 +1,10 @@
 export interface Movie {
   type: 'youtube' | 'vimeo';
   id: string;
-  link: string;
   loading?: boolean;
+  title?: string;
+  publishedAt?: any;
```

Comment:
nie powinieneś igdzie używac explicit any bez przyczyny - a jak masz przyczynę to ją napisz w komentarzu